packages/generator/src/vdm-types.ts

```diff
@@ -80,7 +81,8 @@ export interface VdmNavigationProperty extends VdmPropertyDescriptor {
 export interface VdmEnumType {
   originalName: string;
   typeName: string;
-  members: string[];
+  underlyingType: string;
+  members: Record<string, number | BigNumber>;
```

Comment:
I think it makes sense to introduce a new type for members instead of a `Record`.
Consider:
```ts
export interface VdmEnumMemberType {
  name: string;
  originalValue?: string;
}
```